scico/test/linop/test_radon_svmbir.py

```diff
@@ -25,6 +25,9 @@
 BIG_INPUT = (128, 129, 200, 201)
 SMALL_INPUT = (4, 5, 7, 8)
 
+BIG_INPUT_OFFSET_RANGE = (0, 0.3, 3)
+SMALL_INPUT_OFFSET_RANGE = (0, 0.01, 0.1)
+
```

Comment:
The use of all upper-case variable names is consistent with prior usage in this file, but is inconsistent with scico coding standards.

Reply:
Fixed it

Reply:
Sorry, my mistake. I don't think this usage occurs anywhere else in scico, but @Michael-T-McCann is correct.

Reply:
In `test_radon_astra.py`, global variables such as `rtol_cpu` are in llower case. Should those be changed to uper case as well then?